listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -68,13 +67,14 @@ def get_listens_for_training_model_window(metadata):
     """  Prepare dataframe of listens of X days to train. Here X is a config value.
 
         Returns:
-            training_df (dataframe): Columns can de depicted as:
+            A dataframe with columns as:
                 [
-                    artist_mbids, artist_msid, artist_name, listened_at, recording_mbid,
-                    recording_msid, release_mbid, release_msid, release_name, tags, track_name, user_name
+                    artist_msid, artist_name, listened_at, recording_msid, release_mbid,
+                    release_msid, release_name, tags, track_name, user_name
                 ]
     """
     to_date = datetime.utcnow()
+    to_date = stats.adjust_days(to_date, 5000)
```

Comment:
Is 5000 supposed to be a config value as the comment says?

Reply:
I was experimenting with something, forgot to remove. Thanks :)